src/migrate/azext_migrate/actions.py

```diff
@@ -0,0 +1,107 @@
+import argparse
+from knack.util import CLIError
+
+
+# pylint: disable=protected-access
+
+
+class AddParameters(argparse._AppendAction):
+    def __call__(self, parser, namespace, values, option_string=None):
+        action = self.get_action(values, option_string)
+        super(ImageBuilderAddCustomize, self).__call__(parser, namespace, action, option_string)
+
+    def get_action(self, values, option_string):  # pylint: disable=no-self-use
+        try:
+            properties = dict(x.split('=', 1) for x in values)
+        except ValueError:
+            raise CLIError('usage error: {} [KEY=VALUE ...]'.format(option_string))
+        d = {}
+        for k in properties:
+            kl = k.lower()
+            v = properties[k]
+        return d
+
+
+class AddProject(argparse._AppendAction):
+    def __call__(self, parser, namespace, values, option_string=None):
+        action = self.get_action(values, option_string)
+        super(ImageBuilderAddCustomize, self).__call__(parser, namespace, action, option_string)
+
+    def get_action(self, values, option_string):  # pylint: disable=no-self-use
+        try:
+            properties = dict(x.split('=', 1) for x in values)
+        except ValueError:
+            raise CLIError('usage error: {} [KEY=VALUE ...]'.format(option_string))
+        d = {}
+        for k in properties:
+            kl = k.lower()
+            v = properties[k]
+        return d
+
+
+class AddProject(argparse._AppendAction):
+    def __call__(self, parser, namespace, values, option_string=None):
+        action = self.get_action(values, option_string)
+        super(ImageBuilderAddCustomize, self).__call__(parser, namespace, action, option_string)

```

Comment:
didn't look deep in the code, but `ImageBuilderAddCustomize`?

Reply:
This should have been fixed in the latest code generator